metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache-2.0'
 description      'Installs/Configures Splunk Servers and Forwarders'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.25.0'
+version          '2.25.1'
```

Comment:
This should be 2.26.0 since the change is an enhancement rather than a bug fix.

Reply:
updated the version